### PR TITLE
BUG: Renamed CommentSpellCheck repo

### PR DESCRIPTION
**Pull request by dave3d**

### CODE REVIEW

**Comment by dave3d** [comment]
> Will the new tag work?

Dunno?

File: .github/workflows/PythonLintAndSpell.yml [hunk 1]
```diff
@@ -42,18 +42,18 @@ jobs:
...
       uses: actions/setup-python@v4
       with:
-        python-version: 3.8
+        python-version: 3.9
 
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install git+https://github.com/SimpleITK/SimpleITKSpellChecking@v0.1
+        python -m pip install git+https://github.com/SimpleITK/CommentSpellCheck
```
Comment (blowekamp) [inline .github/workflows/PythonLintAndSpell.yml:53]: Is there a new tag that we could pin?

Comment (dave3d) [reply .github/workflows/PythonLintAndSpell.yml:53]: Yeah, it's currently at release v0.2.0.  I can pin it to that.

**Comment by blowekamp** [comment]
Looks good. Can we squash some of those extra commits before merge?

**Review by blowekamp — APPROVED** [review]
Thanks for the update!

**Pull request merged**